docs/src/sw.js

```diff
@@ -2,7 +2,7 @@
 /* eslint-disable no-restricted-globals */
 /* global workbox */
 
-importScripts('https://storage.googleapis.com/workbox-cdn/releases/4.0.0-alpha.0/workbox-sw.js');
+importScripts('https://storage.googleapis.com/workbox-cdn/releases/4.3.1/workbox-sw.js');
```

Comment:
Did you check the breaking changes. Does it still work?

Reply:
Yeah I'd like a link to the changelog between those versions too. webworker changes are always a bit scary.

Reply:
https://developers.google.com/web/tools/workbox/guides/migrations/migrate-from-v3#workbox-sw

According to this. workbox.skipWaiting has now become workbox.core.skipWaiting and workbox.clientsClaim has become workbox.core.clientsClaim. I think we only need to change self.skipWaiting() to self.core.skipWaiting()
I don't think we need to use self.core.clientsClaim.

Reply:
I'll make the changes

Reply:
We are not using this API.

Reply:
> We are not using this API.

Which API?

Reply:
skipWaiting

Reply:
We need to test the change, but I believe it would work without any change.

Reply:
We can test the change with the Netlify live preview, and by enabling the service work temporary for this branch test.

Reply:
What is self.skipWaiting(); in line 19?

Reply:
> We need to test the change, but I believe it will work without any change.

Okay. So 4.3.1 should not affect the service work according to you?

Reply:
At a confidence rate of 80%, need to test the 20% left.

Reply:
👍

Reply:
If I change only the workbox cdn. And revert date-fns back. Would that work?